### PR TITLE
CIV-9342 add defendant pdf tab docs

### PR DESCRIPTION
**Pull request by kamilewandowski**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-9342

### Change description ###
add defendant response download link

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/services/features/dashboard/claimSummary/latestUpdate/latestUpdateContentBuilder.ts [hunk 1]
```diff
@@ -93,7 +93,7 @@ function getPartAdmitPaidPayImmediately(claim: Claim, lng: string) {
       claimantName: claim.getClaimantFullName(),
     })
     .addParagraph(`${PAGES_LATEST_UPDATE_CONTENT}.WE_WILL_CONTACT_YOU_WHEN_THEY_RESPOND`)
-    .addResponseDocumentLink(`${PAGES_LATEST_UPDATE_CONTENT}.DOWNLOAD_YOUR_RESPONSE`, claim.id, DocumentUri.SEALED_CLAIM)
```
Comment (TatsyMaslova) [inline src/main/services/features/dashboard/claimSummary/latestUpdate/latestUpdateContentBuilder.ts:96]: will not that break getting sealed claim?

File: src/main/routes/features/document/documentDownloadController.ts [hunk 2]
```diff
@@ -7,19 +7,19 @@ import {convertToDocumentType} from 'common/utils/documentTypeConverter';
...
 const documentDownloadController = Router();
 const civilServiceApiBaseUrl = config.get<string>('services.civilService.url');
 
-const civilServiceClientForDocRetrieve: CivilServiceClient = new CivilServiceClient(civilServiceApiBaseUrl, true);
+const civilServiceClientForDocRetrieve: CivilServiceClient = new CivilServiceClient(civilServiceApiBaseUrl);
 
 documentDownloadController.get(CASE_DOCUMENT_DOWNLOAD_URL, async (req: Request, res: Response, next: NextFunction) => {
   try {
-    const claim: Claim = await  getClaimById(req.params.id, req);
+    const claim: Claim = await civilServiceClientForDocRetrieve.retrieveClaimDetails(req.params.id, <AppRequest>req);
```
[the remark this answers is not in the document]
Comment (TatsyMaslova) [reply src/main/routes/features/document/documentDownloadController.ts:18]: if you need a fresh documents it is better to update the claim with them when you are getting fresh claim in claim details page

**Review by laperla11 — DISMISSED** [review]

File: src/main/routes/features/document/documentDownloadController.ts [hunk 3]
```diff
@@ -16,10 +16,11 @@ const civilServiceClientForDocRetrieve: CivilServiceClient = new CivilServiceCli
 
 documentDownloadController.get(CASE_DOCUMENT_DOWNLOAD_URL, async (req: Request, res: Response, next: NextFunction) => {
   try {
-    const claim: Claim = await  getClaimById(req.params.id, req);
+    const claim: Claim = await getClaimById(req.params.id, <AppRequest>req);
```
Comment (TatsyMaslova) [inline src/main/routes/features/document/documentDownloadController.ts:19]: No need to convert to <AppREquest> there

**Review by TatsyMaslova — DISMISSED** [review]

**Review by laperla11 — APPROVED** [review]

**Review by neeta-hmcts — DISMISSED** [review]

File: src/main/routes/features/document/documentDownloadController.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (laperla11) [reply src/main/routes/features/document/documentDownloadController.ts]: reverted this change

**Review by laperla11 — APPROVED** [review]

**Review by TatsyMaslova — APPROVED** [review]

**Pull request merged**